### ISSUE TITLE
Room pills when referenced by their room alias, when a client cannot resolve details about the room, look janky

### ISSUE THREAD

**Issue by ShadowJonathan** [vector-im/element-web#16981]
![image](https://user-images.githubusercontent.com/22740616/115078823-7b287f00-9f00-11eb-82ad-a319ad13e4f9.png)

Maybe it could be better by adding some padding on the right if there is no room 'resolved' for the pill? Normally the avatar would be there.

Or; a new design could be made for detail-less room pills like this.

**Comment by SimonBrandner** [comment]
I think this is basically the first thing on the list [here](https://github.com/vector-im/element-web/issues/4640) but I think it makes sense to keep this issue around for now

**Comment by ShadowJonathan** [comment]
That issue seems to be a bit of a discussion bucket, Matt is throwing a few more issues in, but i think it's useful to "defer" that first todo on the list to this issue? (cross it out and add this issue #)

**Comment by ShadowJonathan** [comment]
Yeah, that first TODO item seems to talk about a comprehensive fix (making room avatars discover-able via the directory and such), while i'm just calling for a temporary visual fix.
